### ISSUE TITLE
C# 9.0 Inheriting the same type of record causes crash

### ISSUE THREAD

**Issue by vsfeedback** [dotnet/roslyn#49263]
_This issue has been moved from [a ticket on Developer Community](https://developercommunity2.visualstudio.com/t/C-90-Inheriting-the-same-type-of-recor/1240188?preview=true)._

---
[severity:It bothers me. A fix would be nice]
When trying to inherit the same type of record causes a crash

Example:

```
...
public record SomeData : [ SomeData ] <- Causes crash
{
    ...
}
...
```
Hope u fix that, would be nice to see

---
#### Feedback Bot on 11/1/2020, 10:58 PM: 

<p>We have directed your feedback to the appropriate engineering team for further evaluation. The team will review the feedback and notify you about the next steps.</p>

#### Feedback Bot on 11/2/2020, 08:07 AM: 

<p>Thank you for sharing your feedback! Our teams prioritize action on product issues with broad customer impact. See details at: <a target='_blank' href="https://docs.microsoft.com/en-us/visualstudio/ide/report-a-problem?view=vs-2019#faq">https://docs.microsoft.com/en-us/visualstudio/ide/report-a-problem?view=vs-2019#faq</a>. In case you need answers to common questions or need assisted support, be sure to use <a target='_blank' href="https://visualstudio.microsoft.com/vs/support/">https://visualstudio.microsoft.com/vs/support/</a>. We’ll keep you posted on any updates to this feedback.</p>

#### Feedback Bot on 11/2/2020, 08:28 AM: 

<p>We have directed your feedback to the appropriate engineering team for further evaluation. The team will review the feedback and notify you about the next steps.</p>

---
### Original Solutions
(no solutions)

**Comment by Youssef1313** [comment]
I don't see a crash in SharpLab, only compile errors:

https://sharplab.io/#v2:EYLgtghgzgLgpgJwDQxAN0QSwGYE8A+AAgMwAECcAxgPYIAmpAytWHACIQwSkhMvucIAWABQAb1EBfUUA===

**Comment by jcouv** [comment]
This was fixed (I think in 16.9). Verified that we have a test for it.
